src/curl.cpp

```diff
@@ -1622,7 +1622,7 @@ bool S3fsCurl::CreateCurlHandle(bool force)
       S3FS_PRN_ERR("could not destroy handle.");
       return false;
     }
-    S3FS_PRN_INFO3("already has handle, so destroied it.");
+    S3FS_PRN_INFO3("already has handle, so destroyed it.");
```

Comment:
Since the destruction occurred before the message, I would say keep the existing tense, `destroyed`.